pytube/__main__.py

```diff
@@ -362,8 +363,24 @@ def description(self) -> str:
         """Get the video description.
 
         :rtype: str
-        """
-        return self.vid_info.get("videoDetails", {}).get("shortDescription")
+        """        # Get the Html file of the page with a timeout of 10 seconds
+        page_detail = request.get(self.watch_url, timeout=10)
+
+        # The start and end point of the description in page
+        start = re.compile(r'shortdescri\w+', re.IGNORECASE)
+        end = re.compile(r'iscrawl\w+', re.IGNORECASE)
+
+        # Finds the pattern in the page
+        match1 = start.findall(page_detail)
+        match2 = end.findall(page_detail)
+
+        if match1 and match2:
+            # Retrives the index and trims not required commas
+            ind1 = page_detail.index(''.join(match1)) + 19
+            ind2 = page_detail.index(''.join(match2)) - 3
```

Comment:
What are these magic numbers?

Reply:
Initially, I considered creating another request and then retrieving the description from the HTML. However, I discovered that "self.watch_html" also contains the description between "Shortdescription" and "iscrawlable." Therefore, I utilized a regex to find their indexes and return anything in between them, which constitutes the description. The addition and subtraction of 3 is for removing the colon and double quotes to get the text only.

The latest commit "Removed Internal request for Description - 2" consist of all the changes i mentioned